x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/space_management/space_management.tsx

```diff
@@ -56,9 +64,26 @@ export const SpaceManagement: FC<Props> = ({ spacesApi, setCurrentTab }) => {
     };
   }, []);
 
+  useEffect(
```

Comment:
if you only want to invoke this effect for initial value 
```suggestion
  useMount(
```

Reply:
As `usePermissionCheck` uses an Observable underneath to load the capabilities, I'd worry that using `useMount` rather than a `useEffect` that looks for changes in the capabilities could possibly cause a race condition where the capabilities aren't ready at the time of the first render.
I'm going to leave this as `useEffect` to avoid this possible outcome.